package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -284,13 +284,16 @@ public class GeometrySubsets {
       Material mat = renderer.sharedMaterial;
       bool changeHandedness = options.changeHandedness == BasisTransformation.SlowAndSafe ||
                               options.changeHandedness == BasisTransformation.SlowAndSafeAsFBX;
+      // check if the handedness needs to change
+      bool isLeftHanded = usdMesh.orientation == Orientation.LeftHanded;
+      var handednessChanged = changeHandedness && !isLeftHanded || !changeHandedness && isLeftHanded;
```

Comment:
For readability, rather than making a new variable, could we just reuse `changeHandedness`?
I would also even kill `isLeftHanded` and just accumulate that into `changeHandedness` as well.

Otherwise when reading this code, you will load three variables into your brain, only to realize that two of them are unused after this line.